modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -14,18 +14,29 @@ settings and run `openshift-install` to install a cluster and `oc` to work with
 for instructions. See link:https://youtu.be/2uLKvB8Z5D0[Securing Automated Decryption New Cryptography and Techniques]
 for a presentation on Tang.
 
-. Add kernel arguments to configure networking when you do the RHCOS installations for your cluster.
-If you miss this step, the second boot will fail.
-For example, to configure DHCP networking, identify `ip=dhcp`
-or set static networking when you add parameters to the kernel command line.
+. To configure networking when you do the RHCOS installations for your cluster:
 
+.. Add the `rd.neednet=1` kernel argument as shown in the `MachineConfig` examples below.
+
+.. Use the `ip=dhcp` kernel argument for DHCP networking.
```

Comment:
I suggest using bullet points for this step and the one that follows it. This is because I think typically you would either configure DHCP or static interfaces. There might be situations where you would configure both. But, the steps as they stand are sequential which I think suggests that you need to do both in order. Instead I would use bullets to suggest a choice:

* Specify the `ip=dhcp` kernel argument to configure DHCP networking.
* Specify the necessary `ip=` kernel arguments to configure static networking.

Reply:
Good point! I did not intend them to be sequential.